src/potentiometer-element.ts

```diff
@@ -81,6 +112,7 @@ export class PotentiometerElement extends LitElement {
         />
         <rect x="5.4" y=".70" width="9.1" height="1.9" fill="#ccdae3" stroke-width=".15" />
         <ellipse
+          filter="url(#outline)"
```

Comment:
How about controlling this filter directly from CSS? then you don't really need the input event listeners and updating the stdDeviation from JS

Reply:
always better - will fix :)